src/components/organisms/CardCertificate/CardCertificate.js

```diff
@@ -0,0 +1,108 @@
+import { html, LitElement } from '@polymer/lit-element';
+import CSS from './_components.card-css';
+import '../../molecules/VerifyButton';
+
+class CardCertificate extends LitElement {
+  constructor () {
+    super();
+    this.logoIsErroring = false;
+    this.onImageLoadError = this.onImageLoadError.bind(this);
+  }
+
+  static get properties () {
+    return {
+      hasCertificateDefinition: Boolean,
+      recipientName: String,
+      certificateTitle: String,
+      issuedOn: String,
+      issueDate: String,
+      issuerName: String,
+      issuerLogo: String,
+      recordLink: String,
+      hideRecordLink: Boolean,
+      hideVerifyButton: Boolean,
+      logoIsErroring: Boolean
+    };
+  }
+
+  onImageLoadError () {
+    this.logoIsErroring = true;
+  }
+
+  organizationName (props) {
+    return html`<p class="buv-o-text-15">${props.issuerName}</p>`;
+  }
+
+  organizationLogo (props) {
+    return html`<img src='${props.issuerLogo}' alt='${props.issuerName}' onerror='${this.onImageLoadError}' class='buv-c-card__img'/>`;
+  }
+
+  _render (props) {
+    const {
+      hasCertificateDefinition,
+      recipientName,
+      certificateTitle,
+      issuedOn,
+      issueDate,
+      issuerName,
+      recordLink,
+      hideRecordLink,
+      hideVerifyButton } = props;
+
+    if (!hasCertificateDefinition) {
+      return null;
+    }
+
+    const titleClass = [
+      'buv-c-card__title',
+      hideRecordLink ? 'buv-c-card__title--no-padding' : ''
+    ].join(' ');
+
+    return html`
+      ${CSS}
+      <section class='buv-c-card'>
+        <div class='buv-c-card__img-wrapper'>
+          ${!this.logoIsErroring ? this.organizationLogo(props) : this.organizationName(props)}
```

Comment:
wait what's the difference between this and the `alt` text?

Reply:
could we handle that from a CSS only perspective? I am a bit reluctant to have this kind of code.

Reply:
The alt text doesn't get rendered correctly on all browsers (Safari), and some will show the broken image icon (Chrome).